### ISSUE TITLE
Warning log messages when sending revenue events

### ISSUE THREAD

**Issue by Matt343** [optimizely/python-sdk#212]
Sending an event with the `revenue` tag set generates a warning log message. It seems to be looking for a numeric `value` tag, even though that tag is not set in the call.

The revenue does appear to be tracked despite the warning, and the `value` tag works as expected with no output.

Optimizely SDK version 3.2.0

Example code:
```
optimizely_client = Optimizely(datafile)
optimizely_client.track(event_key='some_event', user_id='abc123', event_tags={'revenue': 456})

# WARNING:optimizely.logger.NoOpLogger:The provided numeric metric value None is in an invalid format and will not be sent to results.
```

**Comment by aliabbasrizvi** [comment]
Thanks fo reporting the issue @Matt343 
We will take a look and get back to you.

**Comment by aliabbasrizvi** [comment]
GitHub auto closed the issue as the fix is in master. Will close this issue when next release goes out (hopefully within next 3-5 days).

**Comment by aliabbasrizvi** [comment]
@Matt343 this issue was fixed in the latest 3.3.0 release that went out on October 28th. 

Thanks again for reporting this. Going to mark this as closed/fixed.
